content/en/case-studies/7.new-york-public-radio.md

```diff
@@ -0,0 +1,50 @@
+
+## A number of the New York Public Radio frontends were built with Ember. Why did you choose to migrate to Nuxt?
+
+Well Ember is tough to work with because no one knows it. On our engineering team in New York Public Radio history, we've never hired one person who already knew Ember, we had to teach them on the job and we kind of got stuck with it, and the community support isn't great. The learning curve is also very, very high. So, when they hired me to manage the engineering team, my boss at the time wanted to switch everything to Vue. So it became my job to switch everything to Vue and of course I'm going to use Nuxt because we need the SEO benefits for our sites, so we just started doing it very gradually. We had to rebuild our whole design system in Vue. And then we started chipping away at some of our websites.
+
+## What are your favorite Nuxt features?
+
+There's so many features that I love! How everything's built in, you don't have to worry about configuring the build process with Webpack and Babel is in there already, the automatic code splitting and [smart prefetching](/announcements/introducing-smart-prefetching) is amazing. **The first load is fast but then when you navigate between page, it's instant and everyone noticed that and even our users have commented on how fast it is.** Stuff like that already built in makes it amazing to use. 
+
+In addition, even something as simple as managing meta tags, is really easy to do with Nuxt whereas with Ember, it was like you're jumping through all these hoops to figure out how to make that dynamic. Besides that, comparing this to other frameworks I've worked with the community is great, the documentation is great. The modules are awesome: We want to add Google Analytics or authentication, it's just a couple lines of code. You don't have to worry about it. **It really lets us focus on just building cool things and not worrying about the infrastructure**.
+
+## For now you have two projects built with Nuxt, the WNYC player, and the Gothamist.
+
+Yes, the first thing we did was build the [WNYC radio page](https://www.wnyc.org/radio/) in Nuxt, it's a subset of the WNYC site where people can go to listen to the radio online to the FM stream or the AM stream. We have a stream coming in through WebSockets, and we built a couple NPM packages to help with streaming audio services. But besides that, everything else was just built in with Nuxt so it's pretty easy to do.
+
+![The Gothamist article](img/case-studies/nypr/1.png){width=736 height=407}
+
+## For the Gothamist website, did you choose Server Side Rendering or Static Generation?
+
+We chose to do an universal app, because we wanted a single page app as we are planning to add an audio player that's persistent on the bottom, that's the main reason we didn't choose SSG.
```

Comment:
Would be good to get more info on this as I worry it might be confusing for a reader.

Reply:
Good point, I can remove this line from now and edit it when I get more infos.